examples/src/components/utils/ExampleWithExport.jsx

```diff
@@ -14,13 +14,19 @@ export class ExampleWithExport extends React.Component {
         this.state = {
             showExportDialog: false,
             errorMessage: null,
+            exportResult: undefined,
         };
 
         this.doExport = this.doExport.bind(this);
     }
 
+    /**
+     * This will be called by child visualozation once it successfully renders
+     * and its underlying data is ready for export. The child visualization will
+     * call this with one parameter: function that can bre called to trigger the exports.
+     */
     onExportReady = exportResult => {
-        this.exportResult = exportResult;
+        this.setState({ exportResult });
```

Comment:
Why was the move to state necessary?

Reply:
This was suggested by lubko in order to sanitize the code. exportResult is a function that is available after a successful render of the child. Therefore, it should in the state.

Reply:
OK, thanks :)